### DISCUSSION TITLE
设计器的选择项工具菜单允许配置吗？

### DISCUSSION THREAD

**Discussion by hanxinimm** [alibaba/formily#1812] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/13499426/125916973-90170e67-03af-486b-9ef9-34792be8d2c2.png)

我们内部工程编辑，有些的配置和选项作为默认值初始化
比如这个工具菜单，我们希望这些默认的初始化选项不可以编辑，或者不可以排序更换位置，或者可以增加更多自定义按钮操作

**Comment by janryWang** [comment, marked as answer]
自己替换掉Toolbar组件就行，designable是一切可替换的

**Reply by hanxinimm** [reply]
@janryWang  有代码示例吗？没找到应该在哪个地方去替换或者配置

**Reply by janryWang** [reply]
本身playground中使用的组件就是组合模式的，随便你替换
